index.html

```diff
@@ -580,7 +586,7 @@ <h2>Verification Methods</h2>
                   <p>
 The value of the <code>id</code> property for a <a>verification
 method</a> MUST be a <a data-cite="INFRA#string">string</a> that conforms to the
-[[URL]] syntax.
+conforms to the URI syntax as defined in [[RFC3986]].
```

Comment:
This deviates from the use of URL in DID Core -- https://www.w3.org/TR/did-core/#verification-methods

-1 for deviating from DID Core unless we have a very good reason to do so.

Reply:
completely get it - makes sense - do we want to point to RFC3986 and RFC1738 for clarity on rules for resolution and especially for handling relative or other scenarios that could conceivably arise in addition to the whatwg reference?

Reply:
> do we want to point to RFC3986 and RFC1738 for clarity

Not for URL, no. Those are legacy and contain a significant number of bugs that led to the need for the WHATWG URL specification.

If the range here is meant to be a URL, the WHATWG reference is the latest one AFAIK -- https://url.spec.whatwg.org/#urls.

Reply:
> would just 3986 be sufficient?

No, the new URL spec exists specifically because of deficiencies in RFC3986.